CommandHelper/LocalPackages/commands/chars/electro.yml

```diff
@@ -10,11 +10,13 @@ permissions:
   - essentials.fly
   - magic.cast.electrictp
   - magic.commands.cast
+  - magic.cast.electricfloor
 powers:
 - /buff
 - /fly
 - /run <0-2>
 - /cast electrictp
+- /cast electricfloor
```

Comment:
From now on, battle spells will need a wand or item associated with it so that they are easy to use in combat.